Dockerfile

```diff
@@ -46,7 +46,8 @@ RUN svn co https://github.com/docker/docker/branches/$ENGINE_BRANCH/docs/extend
  && jekyll build -s allv -d allvbuild \
  && rm -rf allvbuild/apidocs/layouts \
  && find allvbuild -type f -name '*.html' -print0 | xargs -0 sed -i 's#href="https://docs.docker.com/#href="/#g' \
- && rm -rf allv
+ && rm -rf allv \
+ && apt-get update \
+ && apt-get -y install nginx
 
-# Serve the site, which is now all static HTML
-CMD jekyll serve -s /usr/src/app/allvbuild --no-watch -H 0.0.0.0 -P 4000
+CMD echo "Server running at http://0.0.0.0:4000" && nginx -c /usr/src/app/allvbuild/nginx.conf
```

Comment:
Probably cleaner to copy the nginx.conf to it's default location;

```Dockerfile
COPY nginx.conf /etc/nginx/
```

Also nginx is started in a shell here, therefore not being PID1 (this line will run `/bin/sh -c 'echo .. && nginx -c ...`).

If we are ok to omit the `"Server running at .."`, change to;

```Dockerfile
CMD ["nginx", "-g", "daemon off;"]
```

Otherwise, I _think_ this will work (adding `exec`);

```Dockerfile
CMD echo "Server running at  http://0.0.0.0:4000" && exec nginx -c /usr/src/app/allvbuild/nginx.conf
```

You can verify it's working correctly by running the container, and checking the `nginx` is PID1